packages/categorize/configure/src/design/index.jsx

```diff
@@ -239,8 +240,8 @@ export class Design extends React.Component {
                   categoriesPerRow: categoriesPerRow.settings && numberField(categoriesPerRow.label,
                     {
                           label: categoriesPerRow.label,
-                          min: 1,
-                          max: 4,
+                          min: lowerMaximumValueCategoriesPerRow,
```

Comment:
the name is too long for this one. I think you can rename to `minCategoriesPerChoice` if I understand correctly what it refers to